src/compiler/diagnosticMessages.json

```diff
@@ -2824,14 +2824,25 @@
         "category": "Message",
         "code": 6136
     },
-    "No types specified in 'package.json' but 'allowJs' is set, so returning 'main' value of '{0}'": {
+
+    "List of compiler extensions to require.": {
         "category": "Message",
-        "code": 6137
+        "code": 6150
+    },
+    "Extension loading failed with error '{0}'.": {
+        "category": "Error",
+        "code": 6151
     },
     "Property '{0}' is declared but never used.": {
         "category": "Error",
         "code": 6138
     },
+

```

Comment:
These got merged out-of-order. `6138` should be above them (with an extra line between them to show that there's open numbers between).